spawn.lua

```diff
@@ -1,66 +1,45 @@
+function SendToWorldSpawn( Player )
+
+	local World = Player:GetWorld()
+	Player:TeleportToCoords(World:GetSpawnX(), World:GetSpawnY(), World:GetSpawnZ())
```

Comment:
I took it from https://github.com/cuberite/cuberite/issues/3305 comment by LogicParrot :

>  Oh. Preventing falling through unloaded chunks was added in April.

And from [IRC logs](https://botbot.me/freenode/cuberite/), August 11th, 2016, from LogicParrot :
>Previously, you had to wait for chunk loading prior to teleport
>Otherwise, players may fall through the unloaded chunk
>I fixed this in ~April
>But some plugins still wait for chunk loading prior to teleport
>Because their code simply is not updated.